client/signup/steps/jpo-homepage/index.jsx

```diff
@@ -0,0 +1,120 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import { connect } from 'react-redux';
+import get from 'lodash/get';
+import classNames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import StepWrapper from 'signup/step-wrapper';
+import SignupActions from 'lib/signup/actions';
+import Card from 'components/card';
+import Button from 'components/button';
+import { translate } from 'i18n-calypso';
+import { setJPOHomepage } from 'state/signup/steps/jpo-homepage/actions';
+import StaticGraphic from './static-graphic';
+import NewsGraphic from './news-graphic';
+
+const JPOHomepageStep = React.createClass( {
+	propTypes: {
+		flowName: PropTypes.string,
+		goToNextStep: PropTypes.func.isRequired,
+		positionInFlow: PropTypes.number,
+		setJPOHomepage: PropTypes.func.isRequired,
+		signupProgress: PropTypes.array,
+		stepName: PropTypes.string,
+	},
+
+	onSelectNews() {
+		this.submitStep( 'news' );
+	},
+
+	onSelectStatic() {
+		this.submitStep( 'static' );
+	},
+
+	submitStep( jpoHomepage ) {
+		this.props.setJPOHomepage( jpoHomepage );
+
+		SignupActions.submitSignupStep( {
+			processingMessage: translate( 'Setting up your site' ),
+			stepName: this.props.stepName,
+			jpoHomepage
+		}, [], { jpoHomepage } );
+
+		this.props.goToNextStep();
+	},
+
+	skipStep() {
+		this.props.goToNextStep();
+	},
+
+	renderStepContent() {
+		return (
+			<div className="jpo__homepage-wrapper">
+				<div className="jpo__homepage-row">
+					<a className="jpo-homepage__select-news" href="#" onClick={ this.onSelectNews }>
+						<Card
+							className={ classNames( {
+								'is-selected': 'news' === get( this.props.signupDependencies, 'jpoHomepage', '' )
+							} ) }
+							>
+							<NewsGraphic />
+							<Button onClick={ this.onSelectNews }>{ translate( 'Recent news or updates' ) }</Button>
+							<div className="jpo__homepage-description">
+								{ translate( 'We can pull the latest information into your homepage for you.' ) }
+							</div>
+						</Card>
+					</a>
+					<a className="jpo-homepage__select-static" href="#" onClick={ this.onSelectStatic }>
+						<Card
+							className={ classNames( {
+								'is-selected': 'static' === get( this.props.signupDependencies, 'jpoHomepage', '' )
+							} ) }
+							>
+							<StaticGraphic />
+							<Button onClick={ this.onSelectStatic }>{ translate( 'A static welcome page' ) }</Button>
```

Comment:
Hi! I've found a possible matching string that has already been translated **93** times:
`translate( 'A static page' )` *ES Score: 7*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).